testdata/d2compiler/TestCompile2/vars/basic/shape-label.exp.json

```diff
@@ -92,7 +92,23 @@
                 "range": "d2/testdata/d2compiler/TestCompile2/vars/basic/shape-label.d2,4:4:29-4:5:30",
                 "value": [
                   {
-                    "string": "im a var"
+                    "substitution": {
```

Comment:
does this change make sense to u?

Reply:
Yes, the AST was being modified in place before but now it's not in these cases.

Reply:
ohh ok